tomviz/python/SegmentPores.py

```diff
@@ -360,10 +357,10 @@ def transform_scalars(self, dataset, minimum_radius=0.5, maximum_radius=6.):
                 .GetArrayFromImage(opened)
 
             # temp
-            label_buffer = label_buffer.copy()
-            label_map_dataset = vtkImageData()
-            label_map_dataset.CopyStructure(dataset)
-            utils.set_array(label_map_dataset, label_buffer, isFortran=False)
+            # Transpose the data to Fortran indexing
+            label_buffer = label_buffer.copy().transpose([2, 1, 0])
+            label_map_dataset = dataset.create_child_dataset()
```

Comment:
It looks like the histogram range for this operator gets bigger with these changes. I am wondering if it might be because `dataset.create_child_dataset()` [modifies the spacing](https://github.com/OpenChemistry/tomviz/blob/9a05a7f0dbdc18ff8720f0c58e6063b652b2ecba/tomviz/python/tomviz/utils.py#L186) of the copied dataset. I might be able to add an argument to make the child dataset without modifying the spacing.

I am not really sure if this matters either, though.

Old
===
![segment_pores_old](https://user-images.githubusercontent.com/9558430/66777986-6ea11180-ee98-11e9-8c17-7c3f52fc9143.png)

New
===
![segment_pores_new](https://user-images.githubusercontent.com/9558430/66777989-7365c580-ee98-11e9-9b55-f64f0821a449.png)

Reply:
I found out that the difference between these images is caused by the fact that the spacing does not get transferred from VTK to ITK using the [new method for getting an ITK image](https://github.com/OpenChemistry/tomviz/pull/1958/files#diff-76468b10025d5704a77ba9aa0448ea81R301). If I set the spacing immediately after that line, I get the old behavior.

We might need to figure out the best way to transfer the spacing to the ITK image, especially in the external pipeline case where the data is coming directly from a numpy array rather than a vtkDataObject.

Reply:
This was fixed by adding properties that set and access the spacing in both the internal and external pipelines. The spacing is set on the ITK images, and the results are now the same.